app/src/main/java/org/simple/clinic/drugs/selection/EditMedicinesScreen.kt

```diff
@@ -120,6 +129,13 @@ class EditMedicinesScreen(context: Context, attrs: AttributeSet) : LinearLayout(
 
   private fun doneClicks() = RxView.clicks(doneButton).map { PrescribedDrugsDoneClicked }
 
+  private fun refillMedicineClicks() = RxView.clicks(refillMedicineButton).map { PresribedDrugsRefillClicked }
+
+  override fun showDoneButton() {
+    refillMedicineButton.visibility = GONE
```

Comment:
Can we separate hiding the refill medicine button into its own method, please? Like `hideRefillMedicineButton`.